components/actionbar/index.css

```diff
@@ -12,42 +12,36 @@
-	--spectrum-actionbar-corner-radius: var(--spectrum-corner-radius-100);
+	--spectrum-actionbar-minimum-width: var(--spectrum-action-bar-minimum-width);
+	--spectrum-actionbar-corner-radius: var(--spectrum-corner-radius-medium-size-extra-large);
+	--spectrum-actionbar-spacing-label-to-actiongroup: var(--spectrum-action-bar-label-to-action-group-area);
+	--spectrum-actionbar-spacing-action-group-edge: var(--spectrum-action-bar-edge-to-content-area);
 
 	/* item counter field label */
 	--spectrum-actionbar-item-counter-font-size: var(--spectrum-font-size-100);
 	--spectrum-actionbar-item-counter-line-height: var(--spectrum-line-height-100);
 	--spectrum-actionbar-item-counter-color: var(--spectrum-neutral-content-color-default);
 
 	/* emphasized variation colors */
-	--spectrum-actionbar-emphasized-background-color: var(--spectrum-informative-background-color-default);
-	--spectrum-actionbar-emphasized-item-counter-color: var(--spectrum-white);
+	--spectrum-actionbar-emphasized-background-color: var(--spectrum-neutral-content-color-default);
+	--spectrum-actionbar-emphasized-item-counter-color: var(--spectrum-gray-25);
+	--spectrum-actionbar-emphasized-action-button-label-color: var(--spectrum-gray-25);
+	--spectrum-actionbar-emphasized-icon-color: var(--spectrum-gray-25);
+
+	/* colors - applied to popover */
+	--spectrum-actionbar-popover-background-color: var(--spectrum-background-elevated-color);
+	--spectrum-actionbar-popover-border-color: var(--spectrum-action-bar-border-color);
 
 	/* spacing of action bar bottom and horizontal outer edge */
 	--spectrum-actionbar-spacing-outer-edge: var(--spectrum-spacing-300);
 
-	/* spacing of close button */
-	--spectrum-actionbar-spacing-close-button-top: var(--spectrum-spacing-100);
-	--spectrum-actionbar-spacing-close-button-start: var(--spectrum-spacing-100);
-	--spectrum-actionbar-spacing-close-button-end: var(--spectrum-spacing-75);
-
-	/* spacing of item counter field label */
-	--spectrum-actionbar-spacing-item-counter-top: var(--spectrum-action-bar-top-to-item-counter);
-	--spectrum-actionbar-spacing-item-counter-end: var(--spectrum-spacing-400);
-
-	/* spacing of action group */
-	--spectrum-actionbar-spacing-action-group-top: var(--spectrum-spacing-100);
-	--spectrum-actionbar-spacing-action-group-end: var(--spectrum-spacing-100);
-
 	/* drop shadow */
-	--spectrum-actionbar-shadow-horizontal: var(--spectrum-drop-shadow-x);
-	--spectrum-actionbar-shadow-vertical: var(--spectrum-drop-shadow-y);
-	--spectrum-actionbar-shadow-blur: var(--spectrum-drop-shadow-blur);
-	--spectrum-actionbar-shadow-color: var(--spectrum-drop-shadow-color);
+	--spectrum-actionbar-shadow-horizontal: var(--spectrum-drop-shadow-elevated-x);
+	--spectrum-actionbar-shadow-vertical: var(--spectrum-drop-shadow-elevated-y);
+	--spectrum-actionbar-shadow-blur: var(--spectrum-drop-shadow-elevated-blur);
+	--spectrum-actionbar-shadow-color: var(--spectrum-drop-shadow-elevated-color);
 
```

Comment:
After these drop shadows, I would also suggest setting `--mod-closebutton-icon-color-hover: var(--spectrum-actionbar-emphasized-icon-color);`, so that in dark mode, the emphasized action bar's close button doesn't change its icon color. (it mimics the dark mode default action bar's close button).

https://github.com/user-attachments/assets/07fe23b1-e963-4d94-8dc8-dc3e3e2b1d01